docs/frontend-integration/sdk/sdk-methods.mdx

```diff
@@ -34,13 +34,21 @@ const { transactionHash, stSolAccountAddress } = await solidoSDK.stake({
     amount: 20,
     wallet: walletInstance,
 });
+
+// For PDA accounts
```

Comment:
I think it's incorrect to write `PDA accounts`
Like PDA is a type of addresses
As Solana cookbook says `PDAs are home to accounts...`
Maybe let it just like `for PDAs (Program Derived Addresses)`?

Reply:
done